pepper-apis/src/main/java/org/broadinstitute/ddp/model/study/StudySettings.java

```diff
@@ -9,17 +9,20 @@
     private Long inviteErrorTemplateId;
     private boolean analyticsEnabled;
     private String analyticsToken;
+    private boolean shouldDeleteUnsentableEmails;
 
     @JdbiConstructor
     public StudySettings(
             @ColumnName("umbrella_study_id") long studyId,
             @ColumnName("invite_error_template_id") Long inviteErrorTemplateId,
             @ColumnName("analytics_enabled") boolean analyticsEnabled,
-            @ColumnName("analytics_token") String analyticsToken) {
+            @ColumnName("analytics_token") String analyticsToken,
+            @ColumnName("should_delete_unsentable_emails") boolean shouldDeleteUnsentableEmails) {
```

Comment:
I think you mean shouldDeleteUnsenda**b**leEmails . Applies to all usages of `unsentable` in code